lib/TunesTakeoutWrapper.rb

```diff
@@ -0,0 +1,37 @@
+require 'httparty'
+
+module TunesTakeoutWrapper
+  BASE_URL = "https://tunes-takeout-api.herokuapp.com/v1"
+
+  def self.find(id)
+    #/v1/suggestions/VzoikPLQUk2WS7xp
+    HTTParty.get(BASE_URL + "/suggestions/" + id).parsed_response
+  end
+
+  def self.search(term, limit = 20)
+    HTTParty.get(BASE_URL + "/suggestions/search?query=#{term}&limit=#{limit}").parsed_response
+  end
+
```

Comment:
like that you pulled out term and limit into parameters so that we can clearly see what's going on here, rather than hardcoding